### DISCUSSION TITLE
Lammps： Warnning ：Radial neighbor list length of type 0 is not enough

### DISCUSSION THREAD

**Discussion by tiandijunhao** [deepmodeling/deepmd-kit#108] [Q&A] [answered]
When I use freeze.dp model in lammps, it always has a warning which is " Warnning ：Radial neighbor list length of type 0 is not enough" .  

What's the reason, please

**Comment by njzjz** [comment, marked as answer]
﻿sel in your model, which gives the maximum possible number of neighbors in the cut-off radius, is not enough.

**Comment by tiandijunhao** [comment]
> ﻿sel in your model, which gives the maximum possible number of neighbors in the cut-off radius, is not enough.

Thank you very much. Does this mean that I only need to modify the value of sel, keep the rcut at 6A, and then retrain the model? In addition, if I want this model to be able to simulate a large cell in lammps (for example, 10 * 10 * 10, 2000 atoms in total, fcc structure). Is the higher the sel value, the better?

**Comment by tiandijunhao** [comment]
Thank you very much. Does this mean that I only need to modify the value of sel, keep the rcut at 6A, and then retrain the model? In addition, if I want this model to be able to simulate a large cell in lammps (for example, 10 * 10 * 10, 2000 atoms in total, fcc structure). Is the higher the sel value, the better?

**Comment by njzjz** [comment]
> > sel in your model, which gives the maximum possible number of neighbors in the cut-off radius, is not enough.
> 
> Thank you very much. Does this mean that I only need to modify the value of sel, keep the rcut at 6A, and then retrain the model? In addition, if I want this model to be able to simulate a large cell in lammps (for example, 10 * 10 * 10, 2000 atoms in total, fcc structure). Is the higher the sel value, the better?

No, the model will become slower if you set it too high. It’s recommended to estimate the maximum atom number in the cutoff radius (in your model is 6 angstroms).

**Comment by tiandijunhao** [comment]
thank you very much



---Original---
